server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/mcp/mcpServerConfigConverter.ts

```diff
@@ -58,9 +58,16 @@ export class McpServerConfigConverter {
         }
 
         if (isOci) {
```

Comment:
Seems a little convoluted, should we do something like this to avoid the redundant checks? 
```suggestion
if (isOci) {
    const hasTag = pkg.identifier.includes(':')
    const version = hasTag ? '' : `:${version}`
    const baseRef = pkg.registryBaseUrl ? `${pkg.registryBaseUrl}/` : ''
    
    args.push(`${baseRef}${pkg.identifier}${version}`)
} else {
    args.push(`${pkg.identifier}@${version}`)
}
```

Reply:
fixed for readability